modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/ExportCommerceOrderReportMVCResourceCommand.java

```diff
@@ -73,84 +79,134 @@ protected void doServeResource(
+		}
+
+		if (shippingAddress != null) {
+			orderInfo.put(
+				"shippingAddressCity", shippingAddress.getCity()
+			).put(
+				"shippingAddressCountry",
+				() -> {
+					Country country = shippingAddress.getCountry();
+
+					if (country == null) {
+						return StringPool.BLANK;
 					}
-				).put(
-					"shippingAddressName", shippingAddress.getName()
-				).put(
-					"shippingAddressPhoneNumber",
-					shippingAddress.getPhoneNumber()
-				).put(
-					"shippingAddressRegion",
-					() -> {
-						Region region = shippingAddress.getRegion();
-
-						if (region == null) {
-							return StringPool.BLANK;
-						}
-
-						return region.getName();
+
+					return country.getName(themeDisplay.getLocale());
+				}
+			).put(
+				"shippingAddressName", shippingAddress.getName()
+			).put(
+				"shippingAddressPhoneNumber", shippingAddress.getPhoneNumber()
+			).put(
+				"shippingAddressRegion",
+				() -> {
+					Region region = shippingAddress.getRegion();
+
+					if (region == null) {
+						return StringPool.BLANK;
 					}
-				).put(
-					"shippingAddressStreet1", shippingAddress.getStreet1()
-				).put(
-					"shippingAddressStreet2", shippingAddress.getStreet2()
-				).put(
-					"shippingAddressStreet3", shippingAddress.getStreet3()
-				).put(
-					"shippingAddressZip", shippingAddress.getZip()
-				).build()));
+
+					return region.getName();
+				}
+			).put(
+				"shippingAddressStreet1", shippingAddress.getStreet1()
+			).put(
+				"shippingAddressStreet2", shippingAddress.getStreet2()
+			).put(
+				"shippingAddressStreet3", shippingAddress.getStreet3()
+			).put(
+				"shippingAddressZip", shippingAddress.getZip()
+			).put(
+				"shippingPhone", shippingAddress.getPhoneNumber()
+			);
+		}
+
+		List<CommerceOrderItem> commerceOrderItemList =
+			commerceOrder.getCommerceOrderItems();
+
+		orderInfo.put(
+			"commerceOrderId", commerceOrder.getCommerceOrderId()
+		).put(
+			"logoUrl", _getLogoURL(themeDisplay)
+		).put(
+			"orderDate",
+			(commerceOrder.getOrderDate() != null) ?
+				commerceOrder.getOrderDate() : ""
+		).put(
+			"printedNote",
+			(commerceOrder.getPrintedNote() != null) ?
+				commerceOrder.getOrderDate() : ""
+		).put(
+			"unitsCount", commerceOrderItemList.size()
+		).put(
+			"subtotal", commerceOrder.getTotalMoney()
+		).put(
+			"shippingAmount", commerceOrder.getShippingMoney()
+		).put(
+			"shippingDiscountAmount",
+			_commercePriceFormatter.format(
+				commerceOrder.getCommerceCurrency(),
+				commerceOrder.getShippingDiscountAmount(),
+				themeDisplay.getLocale())
+		).put(
+			"taxAmount",
+			_commercePriceFormatter.format(
+				commerceOrder.getCommerceCurrency(),
+				commerceOrder.getTaxAmount(), themeDisplay.getLocale())
+		).put(
+			"totalWithTax", commerceOrder.getTotalWithTaxAmountMoney()
+		).put(
+			"purchaseOrderNumber", commerceOrder.getPurchaseOrderNumber()
+		).put(
+			"locale", themeDisplay.getLocale()
+		);
+
```

Comment:
@christian-chiappa can you please sort these values alphabetically?

Reply:
@ivicac sorted